pkg/cmd/cli/cmd/factory.go

```diff
@@ -86,19 +81,32 @@ func NewFactory(clientConfig clientcmd.ClientConfig) *Factory {
 
 // Clients returns an OpenShift and Kubernetes client.
 func (f *Factory) Clients(cmd *cobra.Command) (*client.Client, *kclient.Client, error) {
-	os, err := f.OpenShiftClientConfig.ClientConfig()
+	cfg, err := f.OpenShiftClientConfig.ClientConfig()
 	if err != nil {
 		return nil, nil, err
 	}
-	oc, err := client.New(os)
+
+	transport, err := kclient.TransportFor(cfg)
```

Comment:
Note that I need to wrap both the origin *and* the kube clients, so that it works when doing i.e. `osc get <kubeobject>`